### DISCUSSION TITLE
Nova 2 EARLY ACCESS

### DISCUSSION THREAD

**Discussion by lukeocodes** [deepgram/community#349] [News + Announcements]
# Introducing Nova-2 Early Access - our next generation speech-to-text model

At Deepgram, we are constantly pushing the boundaries of what’s possible in Language AI. I’m excited to share our latest model for superhuman speech-to-text, Nova-2.

Nova-2 achieves:
- Next-level accuracy for pre-recorded and real-time audio: 18% more accurate than Nova-1 and 36% more accurate than OpenAI Whisper (large)
- Unmatched speed: Up to 40x faster than alternatives
- Same low costs as Nova: 3-5x lower costs

Link to blog: https://deepgram.com/learn/nova-2-speech-to-text-api

**Comment by bmaluijb** [comment]
Really cool! Will the Json result stay the same (utterances, paragraphs, etc..), as I'm now relying on that in my application.

**Reply by lukeocodes** [reply]
Yes our transcription response never differs between ASR models!

**Comment by JackBarker21** [comment]
Hi,

I would like to test the new nova-2.

What do I need to use in my Python script?

`options = {
    "punctuate": True,
    "diarize": True,
    "model": '',
    "tier": ''
}`

**Reply by lukeocodes** [reply]
You only need to change the model from `nova` to `nova-2-ea`, but in the future `nova` will just default to the latest version!
